tests/test_normalize_ast.py

```diff
@@ -81,8 +81,7 @@ def test_visit_ExtSlice():
 
 
 ############ normalize_ast #############
-# TODO: add a test that shows that ast.fix_missing_locations is in use,
-# it is probably better to take tests for ast.fix_missing_locations from python sources.
+
```

Comment:
What's the point of just deleting this comment?
The `fix_missing_locations` function is still in use in the code, [look](https://github.com/0dminnimda/pynalyser/blob/19784feaf4f0166dbcaef58d058dec2b6d864d95/pynalyser/normalize_ast.py#L34).

Reply:
And what is the reason for removing this function if you want to do this?